### PR TITLE
chore(add logging around ORA submission ZIP)

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** Added lots of temporary logging around the ORA submission ZIP files to resolve "stuck" processes 

JIRA: [EDUCATOR-5878](https://openedx.atlassian.net/browse/EDUCATOR-5878)

Some will be removed in https://openedx.atlassian.net/browse/EDUCATOR-5886

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Review by mattcarter — APPROVED** [review]
LGTM

File: openassessment/data.py [hunk 1]
```diff
@@ -1168,8 +1169,26 @@ def create_zip_with_attachments(cls, file, submission_files_data):
         csvwriter = csv.DictWriter(csv_output_buffer, cls.SUBMISSIONS_CSV_HEADER, extrasaction='ignore')
         csvwriter.writeheader()
 
+        logger.info("Beginning writing to ZIP file")
```
Comment (nsprenkle) [inline openassessment/data.py:1172]: Does the logger give contextual information like [this is coming from ORA attempting to create the submission archive]? I could see this message being a bit vague without it.

Comment (jansenk) [reply openassessment/data.py:1172]: Yeah, I agree that this isn't particularly useful out of context, but in the specific use case of  "hey, does this method get called? Where the heck does execution end?" it has a purpose

Comment (jansenk) [reply openassessment/data.py:1172]: This task is called infrequently enough that there's very very little chance of overlap

Comment (jansenk) [reply openassessment/data.py:1172]: I've got https://openedx.atlassian.net/browse/EDUCATOR-5886 to address post-diagnosis cleanup

**Pull request merged**